website/blog/2020-08/00/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Gardener v1.8.0 Released
+linkTitle: Gardener v1.8.0
+newsSubtitle: August 06, 2020
+type: blog
+publishdate: 2020-08-06
+archivedate: 2020-10-31
+authors:
+- name: Rafael Franzke
+  email: rafael.franzke@sap.com
+  avatar: https://avatars2.githubusercontent.com/u/19169361?s=460&v=4
```

Comment:
Technically, you don't need these, unless you want them overridden. The website build will take the details from the commit author automatically.
```suggestion
```